src/patternfly/components/Title/docs/code.md

```diff
@@ -0,0 +1,16 @@
+## Overview
+When you don't want to use the Content component and just want to use single title styles.
+
+## Usage
+
+| Class | Applied | Outcomete |
+| -- | -- | -- |
+| `.pf-c-title` | `<h1>`,`<h2>`,`<h3>`,`<h4>`,`<h5>`,`<h6>` |  Initiates a title. Always use it with a modifier class. |
+| `.pf-is-xxxxl` | `.pf-c-title` | Modifies for xxxxl title styles |
+| `.pf-is-xxxl` | `.pf-c-title` | Modifies for xxxl title styles |
+| `.pf-is-xxl` | `.pf-c-title` | Modifies for xxl title styles |
+| `.pf-is-xl` | `.pf-c-title` | Modifies for xl title styles |
+| `.pf-is-lg` | `.pf-c-title` | Modifies for lg title styles |
+| `.pf-is-md` | `.pf-c-title` | Modifies for md title styles |
+| `.pf-is-sm` | `.pf-c-title` | Modifies for sm title styles |
+| `.pf-has-margin` | `.pf-c-title`, `.pf-is-*` | adds top and bottom margins, always use with a title size modifier |
```

Comment:
just `.pf-c-title`, not `.pf-is-*`

Reply:
is that clear though? What if it was `.pf-is-h*` there is no margin to remove on just `.pf-c-title`

Reply:
In this case, would it be `.pf-c-title.pf-is-*`

Reply:
Oooor, if there is no `.pf-is-*` should you apply the same margin you would for the default font size (e.g. `pf-is-md`)?

Reply:
I was just thinking about this for list too, maybe we don't need it at all. If you are using `.pf-c-tltle` you will be using one of the size classes as well, I think we can assume that's obvious. What do you think @jgiardino ?

Reply:
Now I'm wondering if `.pf-c-title` should just have all the styles associated with `.pf-is-md`. But I've been making an assumption all along that components would have a default set of styles defined, which consumers can modify as needed. And I've also been assuming that `md` always maps to the default values (meaning I'd either make something bigger, or smaller, but by default it's medium). I could be totally off-track with this thinking though.